src/session.js

```diff
@@ -47,9 +52,17 @@ function tryReadingIframeLocation(iframe) {
             // we can sort of detect this by checking the location of the iframe
             // if the contentDocument object cannot be accessed, its due to a security error.
             // security error will occur when the frame is on a different origin
-            resolve(iframe.contentDocument.location);
+
+            // Heuristic to detect google auth error in the iframe document
+            var textContent = iframe.contentDocument.body.textContent;
```

Comment:
Hm, that is an excellent question. I'm not sure.

This change isn't really related to this domain change so I'm going to ignore this for now, but yeah I will need to review this when attempting to break out the auth again...

Thanks!